src/main/java/ch/hftm/controllers/AlbumController.java

```diff
@@ -54,6 +63,23 @@ private Pane createImagePane(Image image, Pane samplePane) {
         Pane imagePane = new Pane();
         imagePane.setPrefHeight(samplePane.getPrefHeight());
         imagePane.setPrefWidth(samplePane.getPrefWidth());
+        imagePane.setOnMouseClicked(event -> {
+                ImageService imageService = new ImageService();
+                Node source = (Node) event.getSource();
+                Label imageName = (Label) source.lookup("#imageName");
+                Image image2 = imageService.getImageByFileName(imageName.getText());
```

Comment:
Maybe we can rename this into `imageToDelete` which is a bit more obvious than image2

Reply:
Good point. I'll rename it to imageToDelete as suggested.